### PR TITLE
Removing token validation from the express sample

### PR DESCRIPTION
**Pull request by salman90**
Removing token validation in the express sample, we don't recommend validating tokens using an unsupported third-party library like [jsonwebtoken](https://www.npmjs.com/package/jsonwebtoken). We will be using our new token validation feature in MSAL to validate the tokens in the samples. Changes in this PR  are:
- Removing the validation class.
- Removing token validation in the handleRedirect function.
- Removing unnecessary packages in the package.json.

### CODE REVIEW

**Comment by jasonnutter** [comment]
Can you please provide context on why it is being removed?

**Comment by salman90** [comment]
@jasonnutter In our samples, we don't recommend validating tokens using an unauthorized third-party library like [jsonwebtoken](https://www.npmjs.com/package/jsonwebtoken). We will be using our new token validation feature in MSAL to validate the tokens in the samples.

**Review by samuelkubai — APPROVED** [review]
LGTM

File: samples/msal-node-samples/ExpressTestApp/TestApp/README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 This sample demonstrates a Node.js & Express web application that authenticates users against [Azure Active Directory](https://docs.microsoft.com/azure/active-directory/fundamentals/active-directory-whatis) (Azure AD) and obtains [Access Tokens](https://docs.microsoft.com/azure/active-directory/develop/access-tokens) to call the [Microsoft Graph API](https://docs.microsoft.com/graph/overview) (Graph API) and the [Azure Resource Manager API](https://docs.microsoft.com/azure/azure-resource-manager/management/overview) (ARM API), with the help of [Microsoft Authentication Library for Node.js](https://aka.ms/msalnode) (MSAL Node). It uses Redis for caching session and tokens.
 
-See also: [Caching](../../../lib/msal-node/docs/caching.md)
+See also: [Caching](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-node/docs/caching.md)
```
Comment (jasonnutter) [inline samples/msal-node-samples/ExpressTestApp/TestApp/README.md]: Was this link not working?

Comment (jasonnutter) [reply samples/msal-node-samples/ExpressTestApp/TestApp/README.md]: Would prefer to keep the relative link if possible.

Comment (salman90) [reply samples/msal-node-samples/ExpressTestApp/TestApp/README.md]: Updated with the correct relative link

**Review by jo-arroyo — APPROVED** [review]
LGTM

**Pull request merged**